api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
+
+def startdb()
```

Comment:
Thanks! I think `startdb` is OK. I'm still not sure what `c` and `cname` are, so I'd rather a longer name there (shouldn't cost us much). And `serve` is both a longer function and a more generic name, so I think it needs either a longer function name or a doc.

I mostly agree that a short method with a clear name doesn't need docs. But another way I use function names is as an index to the file -- I'd like to be able to scan the list of functions [and their docs, assuming the docs are shorter than the function itself] and know _which_ function I should bother reading more of (or, just call). So even if I can read `serve` and understand it pretty quickly, I'd like to be able to read a name like `serveUiAndApi` -- or like `private serve()` which says "don't bother with me unless you really need to."

I think part of what I was missing was overall file structure. It looks like `register_commands` is where I should start (the registered commands are the public interface to this file), and that calls `serve`, and that calls `startDb`. Is there a notion of private/protected/public in Ruby / did I miss a hint that `serve` and `startDb` are only relevant within this file? (I wasn't sure that they weren't being exposed for other modules to call, in which case I'd look for more verbose names.)

(That's a little rambly, but trying to expose my thought process.)